python/cairo-core/src/cairo_core/maths.cairo

```diff
@@ -536,3 +536,57 @@ func felt252_bit_length{range_check_ptr}(value: felt) -> felt {
 
     return bit_length;
 }
+
+func felt252_to_bits{range_check_ptr}(limb: felt, bits_ptr: felt*, current_len: felt) -> felt {
```

Comment:
```suggestion
func felt252_to_bits{range_check_ptr}(value: felt, len: felt, dst: felt*) -> felt {
```

to keep consistency with `felt252_to_bytes` names - update internal usage of values